### PR TITLE
feat(stepper): add loading state to next button in stepper

### PR DESCRIPTION
**Pull request by majabnils**
# PR Checklist
https://elvia.atlassian.net/wiki/spaces/TEAMATOM/pages/10427498683/Review+prosess

- [ ] Bumpet package?
- [ ] Updated changelog?
- [ ] Correct date in changelog?

## Describe PR briefly:
https://elvia.atlassian.net/browse/LEGO-3434

### CODE REVIEW

File: packages/components/components/elvis-stepper/src/react/StepContent.tsx [hunk 1]
```diff
@@ -38,14 +38,22 @@ export const StepContent: FC<StepContentProps> = function ({
...
-        <PrimaryButton
-          disabled={!isReachable(isForced, currentStep + 1, steps)}
-          onClick={() =>
-            handleStepChange(isReachable(isForced, currentStep + 1, steps) ? currentStep + 1 : currentStep)
-          }
-        >
-          {steps?.[currentStep]?.nextButtonText ?? 'Neste'}
-        </PrimaryButton>
+        {steps?.[currentStep]?.nextButtonState === 'loading' ? (
+          <PrimaryButton isLoading disabled={!isReachable(isForced, currentStep + 1, steps)}>
```
Comment (TomSchrier) [inline packages/components/components/elvis-stepper/src/react/StepContent.tsx:42]: I think we can add this to the button 😄
```
aria-disabled={isLoading}
aria-label={isLoading ? "Laster inn" : ""}
```
I saw it here: https://www.bekk.christmas/post/2023/24/accessible-loading-button

Comment (majabnils) [reply packages/components/components/elvis-stepper/src/react/StepContent.tsx:42]: Nice!

Comment (TomSchrier) [reply packages/components/components/elvis-stepper/src/react/StepContent.tsx:42]: I saw that I had a few typos in the original comment, so I updated it.

**Review by TomSchrier — APPROVED** [review]
LGTM

**Pull request merged**